autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -650,6 +665,121 @@ def test_scale_up_down_on_rcv3_and_clb(self):
                           'count of {1}'.format(num_clb_nodes_after_scale,
                                                 self.min_servers))
 
+    @tags(speed='slow', type='rcv3', rcv3_mimic='pass', working='true')
```

Comment:
I should actually remove that since it was in place to run only this test without touching the other tags.